subscription-engine/kafka-ws/config/default.conf

```diff
@@ -7,12 +7,14 @@ app.connection.retry.timeout:integer=3
 morgan.log.format:string[]=[":remote-addr","-",":remote-user",":id","':method :url HTTP/:http-version'",":status",":res[content-length]",":referrer",":user-agent"]
 
 # LOGGER
-logger.verbose:boolean=false
-logger.transports.console.level=info
-
-logger.transports.file.enable:boolean=false
-logger.transports.file.level=info
-logger.transports.file.filename=kafka-ws-logs-%DATE%.log
+log.verbose:boolean=false
+log.console.level=info
+log.file.enable:boolean=false
+log.file.level=info
+log.file.filename=kafka-ws-logs-%DATE%.log
+
+# Lightship
+lightship.detect.kubernetes:boolean=false
```

Comment:
The SDK already does this: 
https://github.com/dojot/dojot-microservice-sdk-js/blob/master/lib/serviceStateManager/ServiceStateManager.js#L29

What would be the reason for boot up the lightship server on a random port (when the config is equal to `true`)?

Reply:
It is passing the default value to create the `lightship` config object, so if you want to change any configuration from it, you use the ConfigManager capabilities (like env var parsing and .conf files).

Reply:
But what would be the reason for boot up the lightship server on a random port?

Reply:
I don't know if it would be changed, but it doesn't matter; by defining at least one config for lightship in the `default.conf` file, the ConfigManager will always create an object for lightship. This opens up the possibility of passing another lightship configurations, like changing the port (it will be used by sidecar to prevent conflicts) via the `KAFKA_WS_LIGHTSHIP_PORT` env var, changing the graceful shutdown timeout via the `KAFKA_WS_LIGHTSHIP_GRACEFUL_SHUTDOWN_TIMEOUT` env var etc.